### ISSUE TITLE
feature : metadata wildcard for numeric timezone

### ISSUE THREAD

**Issue by fpeyraud** [ElvishArtisan/rivendell#765]
Hi,
I was wondering if we could extract the numeric timezone information for metadata wildcard replacement to be able to have ISO (`2006-08-14T02:34:56-06:00`) or RFC-3339 (`2006-08-14T02:34:56-06:00`) compliant dates, which are useful for API calls. In the `date` command, it is `%z` and its variations. `z` is already used in Rivendell's metadata wildcards, but maybe `t`, `tt`, `ttt` and `tttt` could act as `%z`, `%:z`, `%::z` and `%:::z` do in `date` command.

**Comment by ElvishArtisan** [comment]
Makes sense, and those TZ fields are already present in date-time values in the WebAPI.

**Comment by ElvishArtisan** [comment]
Implemented the first part of this in [filepath wildcards](http://opsguide.rivendellaudio.org/html/appendix.filepath_wildcards.html) in ba5fe96. Since filepath wildcards are explicitly aligned to follow the usage of ```date(1)``` wherever possible, we were able to use the customary '%z|%:z|%::z' expressions.

The other half of this, adding it to the [metadata wildcards](http://opsguide.rivendellaudio.org/html/appendix.metadata_wildcards.html#sect.metadata_wildcards), will be a bit trickier as datetimes there utilize the '%dt(&lt;dt&gt;)' construct, where &lt;dt&gt; is a series of the [date-time string formatters recognized by Qt](https://doc.qt.io/qt-5/qtime.html#toString). Time zone support there is quite sketchy, essentially being limited to the 't' wildcard, which returns some poorly-specified _soi-disant_ 'timezone' string. UTC offsets do not seem to be supported at all. We could extend it with the ```tt```, ```ttt``` and ```tttt``` strings as you suggest, but I think chances are good that Qt may end up extending that format at some point in future too, likely in an incompatible fashion. Perhaps choose a letter that is not in use at all (something like ```kk```, ```kkk``` and ```kkkk```)?
